research/picai/fl_nnunet/nnunet_client.py

```diff
@@ -329,6 +334,18 @@ def setup_client(self, config: Config) -> None:
                 device=self.device,
             )
 
+            # Need to modify num_epochs before initializing so that LRScheduler
+            # recieves the right number of epochs
+            # This will anneal LR to near zero each round
+            # Default nnunet behaviour is do decrease LR every 250 steps
+            # Maybe LRScheduler should just be an argument
+            local_epochs, local_steps, _, _ = self.process_config(config)
```

Comment:
I might be missing something here, but i think you did this extraction already on line 326?